supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/api/ActionBuilder.scala

```diff
@@ -64,7 +64,9 @@ object ActionBuilder {
     val ContentTypeTextHeader = s"Content-Type: $contentTypeText"
   }
 
-  val cloudSdkImage = "google/cloud-sdk:slim"
+  // TODO revert this to google/cloud-sdk:slim once latest is unbroken
+  // TODO https://github.com/GoogleCloudPlatform/gsutil/issues/806
+  val cloudSdkImage = "google/google/cloud-sdk:251.0.0-slim"
```

Comment:
unrelated to the `ExecutionStore` changes, this just happened to break at the time this work began